Makefile: change LDFLAGS order on the compile line for --static

Signed-off-by: Michael Rommel <rommel@layer-7.net>

--- aiccu_20070115.orig/unix-console/Makefile	2007-01-15 12:04:27.000000000 +0100
+++ aiccu_20070115/unix-console/Makefile	2013-09-29 22:54:42.742963861 +0200
@@ -144,7 +144,7 @@ endif
 all: aiccu
 
 aiccu:	$(OBJS) ${SRCS} ${INCS}
-	$(CC) $(CFLAGS) $(LDFLAGS) -o $@ $(OBJS)
+	$(CC) $(CFLAGS) -o $@ $(OBJS) $(LDFLAGS)
 ifeq ($(shell echo $(CFLAGS) | grep -c "DEBUG"),0)
 ifeq ($(shell echo "$(RPM_OPT_FLAGS)" | wc -c),1)
 	strip $@
